test/fixtures/test-helpers/input.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  app: {
+    'app.js': '// app',
+    'resolver.js': '// resolver'
+  },
+  config: {
+    'environment.js': '"ENV"'
+  },
+  tests: {
+    helpers: {
+      'resolver.js': 'import Resolver from "../../app/resolver";',
```

Comment:
Is this import really correct? From https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/tests/helpers/resolver.js i instead see `import Resolver from '../../resolver';` which i remember i've tried and transforms correctly